onebusaway-android/src/main/java/org/onebusaway/android/ui/HomeActivity.java

```diff
@@ -2018,4 +2025,28 @@ private boolean isSlidingPanelCollapsed() {
     public ArrivalsListFragment getArrivalsListFragment() {
         return mArrivalsListFragment;
     }
+
+    private void setupLogUploadTask() {
```

Comment:
Let's try moving the setupLogUploadTask() method after the user responses to the feedback with a yes or no answer.

Reply:
We have moved the `setupLogUploadTask()` method after the user responses are processed by `FeedbackActivity` or `FeedbackReceiver`